gridt/src/app/settings/settings.page.html

```diff
@@ -0,0 +1,177 @@
+<ion-header>
+  <ion-toolbar >
+    <ion-buttons slot="start">
+      <ion-menu-button></ion-menu-button>
+    </ion-buttons>
+    <ion-title>
+      Settings
+    </ion-title>
+  </ion-toolbar>
+</ion-header>
+
+
+<ion-content padding>
+  <!-- Refreshes page if pulled down (mobile only)-->
+  <ion-refresher slot="fixed" (ionRefresh)="refreshPage($event)" class="ion-hide-md-up">
+    <ion-refresher-content></ion-refresher-content>
+  </ion-refresher>
```

Comment:
Cool! UX wise: how can we make users aware of this feature?

Reply:
I was thinking about maybe a special notification for mobile devices. But that I want to do in a future Issue.